### PR TITLE
Connect Explorer Nextra E2E

### PR DESCRIPTION
**Pull request by martykan**
## Description

Make connect-popup E2E tests work with the new Nextra Explorer. The old Explorer tests remain. 

Currently only web is supported, not webextension. 
It shouldn't be a problem to add support to it, but currently it's too flaky in the old tests, so we wouldn't have a good baseline.

### CODE REVIEW

**Comment by martykan** [comment]
@SocketSecurity ignore-all

File: .github/workflows/connect-nextra-dev-release.yml [hunk 1]
```diff
@@ -108,3 +109,64 @@ jobs:
...
+      run-webextension: true
+
+  passphrase:
+    needs: [build-deploy]
+    uses: ./.github/workflows/template-connect-popup-nextra-test-params.yml
+    with:
+      test-name: passphrase.test
+      DEV_SERVER_HOSTNAME: dev.suite.sldev.cz
+      # currently flaky, run only in nightly
+      run-webextension: ${{ github.event_name == 'schedule' }}
```
Comment (mroz22) [inline .github/workflows/connect-nextra-dev-release.yml:137]: total nitpick: I'd maybe remove all webextension mentions here if it is not supported in the template..

**Review by mroz22 — APPROVED** [review]
👍

File: .github/workflows/connect-nextra-dev-release.yml (hunk 1 above)
Comment (martykan) [reply .github/workflows/connect-nextra-dev-release.yml:137]: I kept them since I presume it will be eventually added to the template

**Pull request merged**